inference-engine/src/mkldnn_plugin/nodes/region_yolo.cpp

```diff
@@ -199,8 +212,19 @@ struct jit_uni_logistic_kernel_f32 : public jit_uni_logistic_kernel, public jit_
                 uni_vmovups(op, vmm_dst);
                 break;
             case InferenceEngine::Precision::BF16:
-                vcvtneps2bf16(ymm_dst, vmm_dst);
-                uni_vmovups(op, ymm_dst);
+                if (mayiuse(avx512_core_bf16)) {
+                    vcvtneps2bf16(ymm_dst, vmm_dst);
+                } else {
+                    std::vector<size_t> in_idxs;
+                    in_idxs.push_back(vmm_dst.getIdx());
+                    std::vector<size_t> aux_idxs;
+                    aux_idxs.push_back(get_aux_vmm(0).getIdx());
+                    aux_idxs.push_back(get_aux_vmm(1).getIdx());
+                    std::vector<size_t> out_idxs;
+                    out_idxs.push_back(ymm_dst.getIdx());
+                    bf16_emu_emitter->emit(in_idxs, out_idxs, aux_idxs);
```

Comment:
As we discussed offline. Can be implemented like this:
```suggestion
bf16_emu_emitter->emit({vmm_dst.getIdx()}, {vmm_dst.getIdx()});
```

Reply:
Done